test/integration/auth.spec.ts

```diff
@@ -356,6 +395,11 @@ describe('admin.auth', () => {
       .should.eventually.be.rejected.and.have.property('code', 'auth/user-not-found');
   });
 
+  it('getUserByProviderId() fails when called with a non-existing federated id', () => {
```

Comment:
Add unit tests for invalid provider id/uid here as well?

Reply:
I think they're already covered by the unit tests. Specifically, these ones:
'should be rejected given no federated id'
'should be rejected given an invalid federated id'
'should be rejected given an invalid federated uid'

(Or have I missed something?)

Reply:
Sorry, I meant to ask for unit tests in test/unit/auth/auth-api-request.spec.ts.

Reply:
Oh; I hadn't noticed that. Done.

(More thought required, but I suspect some of those tests may be redundant since we're already effectively testing them from the api level. It might be possible to eliminate some of them.)